Deploy actions are broken " invalid bucket "policyengine-api-beta-buildlogs""

[deploy is failing](https://github.com/PolicyEngine/policyengine-api-v2/actions/runs/13590643153/job/37995678375):
```
rm -rf artifacts/staging
mkdir -p artifacts/staging/libs
cp -R ../../libs/* artifacts/staging/libs/
#Environment variables are defined in the github worfklow and have the TF_VAR prefix
#because they are also used by terraform and loaded as variables.
#--gcs_log_dir is required because the github deployment role does not have access to the default bucket.
gcloud builds submit --region=us-central1 --tag us-central1-docker.pkg.dev/policyengine-api-beta/api-v2/policyengine-api-full:b32a03860bffbe53e3af02e0de5e9ae63abd03eb --gcs-log-dir gs://policyengine-api-beta-buildlogs
Creating temporary archive of 53 file(s) totalling 828.6 KiB before compression.
Uploading tarball of [.] to [gs://policyengine-api-beta_cloudbuild/source/1740754498.768087-4e16683d0d074d449443e0de066c60ee.tgz]
ERROR: (gcloud.builds.submit) FAILED_PRECONDITION: invalid bucket "policyengine-api-beta-buildlogs"; service account 811022364438-compute@developer.gserviceaccount.com does not have access to the bucket
make[1]: *** [Makefile.deploy:8: deploy] Error 1
make[1]: Leaving directory '/home/runner/work/policyengine-api-v2/policyengine-api-v2/projects/policyengine-api-full'
make: *** [Makefile.deploy:2: publish-docker] Error 2
```

To do any deployment this must have worked at one time (will check to see if we had successful deployments) so maybe something changed administratively.

Alternately we need to set up the beta account properly in the first place.

I am now running bootstrap to create a new beta project we can deploy to
1. ``gcloud init`` to log into my policyengine account.
2. ``cd terraform/project-policyengine-api/`` ``make bootstrap_beta``
  * got organization ID of policyengine.org from the GCP console
  * got billing account id from finding the associated billing account with the existing API
  * got repo owner I'd by using the GitHub API : https://api.github.com/users/policyengine
  * and the repo name itself of course is this one: "PolicyEngine/policyengine-api-v2"

It looks like I don't have permissions required to create a new project associated with that billing account.
```
module.project.module.project-factory.random_id.random_project_id_suffix: Creating...
module.project.module.project-factory.random_id.random_project_id_suffix: Creation complete after 0s [id=e3U]
module.project.module.project-factory.google_project.main: Creating...
╷
│ Error: failed pre-requisites: missing permission on "billingAccounts/***": billing.resourceAssociations.create
│ 
│   with module.project.module.project-factory.google_project.main,
│   on .terraform/modules/project/modules/core_project_factory/main.tf line 73, in resource "google_project" "main":
│   73: resource "google_project" "main" {
│ 
╵
make: *** [Makefile:6: bootstrap_beta] Error 1
```

--- a/.github/workflows/pr.yml
+++ b/.github/workflows/pr.yml
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+
 env:
   TF_BACKEND_bucket: ${{ vars.PROJECT_ID }}-state
   #TF_VAR_whatever will be picked up as terraform variables.
@@ -11,17 +12,17 @@ env:
   TF_VAR_billing_account: ${{ secrets.BILLING_ACCOUNT }}
   TF_VAR_github_repo_owner_id: ${{ github.repository_owner_id }}
   TF_VAR_github_repo: ${{ github.repository }}
-  TF_VAR_project_id: $${{ vars.PROJECT_ID }}
+  TF_VAR_project_id: ${{ vars.PROJECT_ID }}
   TF_VAR_region: ${{ vars.REGION }}
-  TF_VAR_container_tag: ${{ github.sha }}
+  TF_VAR_full_container_tag: ${{ github.sha }}
+  TF_VAR_simulation_container_tag: ${{ github.sha }}
   BUILD_TAG: ${{ github.run_id }}.${{ github.run_number }}.${{ github.run_attempt }}
   COMMIT_TAG: ${{ github.sha }}
-  PROJECT_ID: ${{ vars.PROJECT_ID }}
 jobs:
   build:
     # Any runner supporting Node 20 or newer
-    name: Build and test
     runs-on: ubuntu-latest
+    environment: beta
 
     # Add "id-token" with the intended permissions.
     permissions:
@@ -29,7 +30,7 @@ jobs:
       id-token: "write"
 
     steps:
-    - name: Checkout repo
+    - name: checkout repo
       uses: actions/checkout@v4
     - name: Install uv
       uses: astral-sh/setup-uv@v5
@@ -39,47 +40,5 @@ jobs:
         python-version: "3.11"
     - name: Set up poetry
       run: uv pip install poetry --system
-    - uses: "google-github-actions/auth@v2"
-      with:
-        project_id: ${{ vars.PROJECT_ID }}
-        credentials_json: ${{ secrets.GCP_SA_KEY }}
-    - name: "Set up Cloud SDK"
-      uses: "google-github-actions/setup-gcloud@v2"
-      with:
-        version: ">= 363.0.0"
-    - name: Build and test
+    - name: Build application
       run: make build
-  test_deploy:
-    runs-on: ubuntu-latest
-    name: Test deployment
-    environment: beta
-    env:
-        TF_VAR_stage: beta
-        TF_VAR_is_prod: false
-    
-    permissions:
-      contents: "read"
-      id-token: "write"
-
-    steps:
-    - name: Checkout repo
-      uses: actions/checkout@v4
-    - name: Install uv
-      uses: astral-sh/setup-uv@v5
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: "3.11"
-    - name: Set up poetry
-      run: uv pip install poetry --system
-    - uses: "google-github-actions/auth@v2"
-      with:
-        credentials_json: ${{ secrets.GCP_SA_KEY}}
-        project_id: ${{ vars.PROJECT_ID }}
-    - uses: hashicorp/setup-terraform@v3
-    - name: "Set up Cloud SDK"
-      uses: "google-github-actions/setup-gcloud@v2"
-      with:
-        version: ">= 363.0.0"
-    - name: Test deployment
-      run: make -f Makefile.deploy plan-deploy TAG=${{ github.sha }} PROJECT_ID=${{ vars.PROJECT_ID }}

